molecule_hetznercloud/cookiecutter/scenario/driver/hetznercloud/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -53,6 +52,31 @@
       retries: 300
       with_items: "{{ server.results }}"
 
+    - name: Create volume(s)
```

Comment:
Question: so we don't need to add this before the `Create molecule instance(s)` task? I am wondering because I am not sure the volume then gets attached to the instance? Or it has to happen in this order in order to create and then attach it?

Reply:
Personally I prefer it in this order because the Ansible module otherwise **requires** to specify a location, see https://docs.ansible.com/ansible/latest/collections/hetzner/hcloud/hcloud_volume_module.html. The volume will be created and auto-attached to the existing instance by the given `server` attribute.